specification/validated-relationships-service-api.yaml

```diff
@@ -317,24 +319,44 @@ components:
                   type: string
                   description: The patient's NHS number.
                   example: "9000000001"
+        subject: 
+          type: object
+          description: The person who is the subject.
+          properties:
+            type:
+              type: string
+              enum: [Patient] # Here: https://simplifier.net/packages/hl7.fhir.r4.core/4.0.1/files/81189/~details this looks like it should be a URI
```

Comment:
Leaving comment here because can't comment on an unchanged line, but line 310, the source of the QuestionnaireResponse could also be a Patient (i.e. when the patient nominates), i.e. enum: [ RelatedPerson, Patient]